framework_tvb/tvb/core/adapters/abcdisplayer.py

```diff
@@ -183,8 +195,22 @@ def dump_with_precision(xs, precision=3):
         format_str = "%0." + str(precision) + "g"
         return "[" + ",".join(format_str % s for s in xs) + "]"
 
-    def _load_h5_of_gid(self, entity_gid):
+    def load_h5_of_gid(self, entity_gid):
```

Comment:
Because it is used outside of it's class in several places

Reply:
That's ok, I missed it

Reply:
I think this method deserves a TODO in all its usages. Can't we avoid using it, as it exposes too much of the internal structures in viewers? For example, small objects (like Volume hasTraits) could be loaded entirely (with the generic api) instead of having to use the file path and h5class in the viewer.
For larger objects (surface, timeseries) maybe we could have helper methods on these datatype classes, and also avoid using the file path in the viewer